crates/occt-sys/src/lib.rs

```diff
@@ -1,9 +1,46 @@
-use std::path::Path;
+use std::{
+    env::var,
+    path::{Path, PathBuf},
+};
+
+const LIB_DIR: &str = "lib";
+const INCLUDE_DIR: &str = "include";
 
 /// Get the path to the OCCT library installation directory to be
 /// used in build scripts.
 ///
 /// Only valid during build (`cargo clean` removes these files).
-pub fn occt_path() -> &'static Path {
-    Path::new(env!("OCCT_PATH"))
+pub fn occt_path() -> PathBuf {
+    // moves the output into target/TARGET/OCCT
+    // this way its less likely to be rebuilt without a cargo clean
+    Path::new(&var("OUT_DIR").expect("missing OUT_DIR")).join("../../../../OCCT")
+}
+
+/// Build the OCCT library.
+pub fn build_occt() {
```

Comment:
Nobody call this (as of this PR), is that intended?

Reply:
I believe there's a part 2 coming for this PR. We first wanted to merge this to publish the next version of occt-sys since it is not included in the workspace.